### PR TITLE
MODELIX-814 Newly created child nodes aren't properly persisted in MPS

### PR DESCRIPTION
**Pull request by mhuster23**

### CODE REVIEW

File: bulk-model-sync-lib/src/commonMain/kotlin/org/modelix/model/sync/bulk/ModelImporter.kt [hunk 1]
```diff
@@ -224,7 +224,9 @@ class ModelImporter(
                     val existingNodeReference = originalIdToExisting[expectedId]
                     if (existingNodeReference == null) {
                         val newChild = existingParent.addNewChild(role, newIndex, expectedConcept)
-                        newChild.setPropertyValue(NodeData.idPropertyKey, expectedId)
+                        if (newChild.originalId() == null) {
```
Comment (odzhychko) [inline bulk-model-sync-lib/src/commonMain/kotlin/org/modelix/model/sync/bulk/ModelImporter.kt:227]: nit: Can this be sensibly teste? Maybe in `bulk-model-sync-lib-mps-test`?

File: mps-model-adapters/src/main/kotlin/org/modelix/model/mpsadapters/MPSLanguageRepository.kt [hunk 2]
```diff
@@ -39,8 +38,6 @@ data class MPSLanguageRepository(private val repository: SRepository) : ILanguag
 
         val conceptDescriptor = ConceptRegistry.getInstance().getConceptDescriptor(conceptId)
 
-        if (conceptDescriptor is IllegalConceptDescriptor) return null
```
Comment (odzhychko) [inline mps-model-adapters/src/main/kotlin/org/modelix/model/mpsadapters/MPSLanguageRepository.kt:42]: Is this fixing the test added in 19cccb9f? (If yes, could we combine both commits to make it more clear?)

File: bulk-model-sync-lib/src/commonMain/kotlin/org/modelix/model/sync/bulk/ModelImporter.kt (hunk 1 above)
Comment (mhuster23) [reply bulk-model-sync-lib/src/commonMain/kotlin/org/modelix/model/sync/bulk/ModelImporter.kt:227]: This is covered by the new test case. That's how I discovered the issue.

File: mps-model-adapters/src/main/kotlin/org/modelix/model/mpsadapters/MPSLanguageRepository.kt (hunk 2 above)
Comment (mhuster23) [reply mps-model-adapters/src/main/kotlin/org/modelix/model/mpsadapters/MPSLanguageRepository.kt:42]: No, this is WIP but might be a necessary step towards the solution.

**Review by languitar — APPROVED** [review]
In terms of functionality this works in the client project.

**Pull request merged**